bibliopixel/__init__.py

```diff
@@ -15,12 +16,23 @@
     from . layout import matrix_drawing as matrix
     from . layout.geometry.rotation import Rotation
 
+__version__ = '3.4.18'
+__status__ = 'Development'
 
-def _get_version():
-    from os.path import abspath, dirname, join
-    filename = join(dirname(abspath(__file__)), 'VERSION')
-    return open(filename).read().strip()
+__description__ = """
+BiblioPixel is a pure Python 3 library for all your LED animations needs. Through its fully output agnostic design you can write your code once and use it on a huge variety of outputs, from LED strips to cubes and even a high performance LED simulator!
 
+BiblioPixel allows many different display geometries. Many of the existing hardware models are LED strips, matrixes or circles, but there are no real limits on what the output device can be, due to the output drivers model.
+"""
+__author__ = 'Adam Haile, Tom Swirly'
```

Comment:
My actual name is Tom Ritchford.  :-D  Tom Swirly is my stage name, which has however leaked into my Git Hub profile.

Reply:
You learn something new every day!